db/migrate/20191118151240_migrate_profile_data.rb

```diff
@@ -0,0 +1,20 @@
+class MigrateProfileData < ActiveRecord::Migration[5.2]
+  def up
+    User.all.each do |user|
+      next if user.users_social_profile.present?
+      next if user.provider.blank?
+
+      puts "Migrating user #{user.name} for #{user.provider}"
```

Comment:
I like a hint about what stuff changed on migration, but happy to remove if it's not your style

Reply:
I like this too; keep it ;-)